pkg/plugins/client.go

```diff
@@ -130,7 +130,7 @@ func (c *Client) callWithRetry(serviceMethod string, data io.Reader, retry bool)
 				return nil, err
 			}
 			retries++
-			logrus.Warnf("Unable to connect to plugin: %s, retrying in %v", req.URL, timeOff)
+			logrus.Warnf("Unable to connect to plugin: %s:%s, retrying in %v", req.URL.Host, req.URL.Path, timeOff)
```

Comment:
This just makes the warning output human readable, otherwise the URL is escaped (and I did't want to use `url.QueryUnescape` since that's introducing a potential error and this was good enough).